app/partials/app.cjsx

```diff
@@ -19,9 +19,12 @@ PanoptesApp = React.createClass
     geordi: @geordiLogger
 
   getEnv: ->
+    shell_env = process.env.NODE_ENV
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env?[1] || 'staging'
+    env = shell_env || browser_env?[1] || 'staging'
+    @setState env: env
```

Comment:
why is this using setState? the result of @getEnv() in `getInitialState` will set this anyway.